.github/workflows/deploy_to_environment_azure.yml

```diff
@@ -121,51 +132,12 @@ jobs:
               --image ${{ env.ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ env.DOCKER_IMAGE_TAG }} \
               --output none
 
-  # This has been disabled so that existing Github workflow tags are not overwritten
```

Comment:
Why are you removing this? I thought it was going to go back in when we started using the new deployment mechanism.

Reply:
If you're happy to continue using this strategy the new deployment mechanism then we can merge it back in, I didn't want to include unused code in this PR

Reply:
Ok, sounds good. Thanks @DrizzlyOwl.